### DISCUSSION TITLE
请问有没有给 没有x,y位置的节点 批量自动设定位置/排序的方法/插件？

### DISCUSSION THREAD

**Discussion by PineSongCN** [antvis/X6#4434] [Q&A] [answered]
我有一些有父子关系的数据，需要展示为横向树形图

**Comment by zlecheng** [comment]
布局呗

**Reply by PineSongCN** [reply]
> 布局呗

刚搜到了布局页面，但是里面的预览之类的都404了
https://x6.antv.antgroup.com/temp/layout#dagre

**Reply by zlecheng** [reply]
> > 布局呗
> 
> 刚搜到了布局页面，但是里面的预览之类的都404了 https://x6.antv.antgroup.com/temp/layout#dagre

你这个地址就看看布局参数就行了

**Comment by zlecheng** [comment, marked as answer]
https://x6.antv.antgroup.com/examples/layout/tree/#mindmap   看这里的demo

**Reply by PineSongCN** [reply]
> https://x6.antv.antgroup.com/examples/layout/tree/#mindmap 看这里的demo

感谢大佬

**Comment by resetsix** [comment]
如果你只是想要一些展示效果注重展示 node 之间的关系，推荐使用[G6 关系图](https://g6-next.antv.antgroup.com/)，有非常丰富的[布局类型](https://g6-next.antv.antgroup.com/manual/core-concept/layout)。

而 X6 更注重数据血缘，流程编排，ER 图等工作流场景。

**Reply by PineSongCN** [reply]
> 如果你只是想要一些展示效果注重展示 node 之间的关系，推荐使用[G6 关系图](https://g6-next.antv.antgroup.com/)，有非常丰富的[布局类型](https://g6-next.antv.antgroup.com/manual/core-concept/layout)。
> 
> 而 X6 更注重数据血缘，流程编排，ER 图等工作流场景。

嗯嗯，X6可以使用React节点，更灵活一些
